### ISSUE TITLE
Join banner on landing page visible during contribution phase

### ISSUE THREAD

**Issue by ryancreatescopy** [ethereum/clrfund#126]
In our round there will be a join phase prior to contributions being open. On develop branch when I deploy the round it starts during the contribution phase. During this phase I can still add projects and the Join banner on the landing page is visible. 

@samajammin just wanted to check that this is a known thing and I hope this serves as a reminder that we may need to update the logic of the banner on the homepage to disappear during contribution phase. Atm it disappears during reallocation.

The banner in question:
![image](https://user-images.githubusercontent.com/40891631/121317750-81384c00-c902-11eb-85f6-2f644ba1465d.png)

- [x] Fix banner logic
- [x] Update to dynamic time remaining (https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/Landing.vue#L18)

**Comment by samajammin** [comment]
@ryancreatescopy good catch 👍 

Currently the conditional is based on `!$store.getters.hasContributionPhaseEnded`: https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/Landing.vue#L27

It should be based on `$store.getters.isRoundJoinPhase`.

**Comment by samajammin** [comment]
@ryancreatescopy is it intentional that mobile & desktop have different content?
See https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/Landing.vue#L17
```
        <div v-if="!$store.getters.hasContributionPhaseEnded && !isRoundFull" class="apply-callout mobile">
          <div id="countdown" class="caps">11 days</div>
          <div id="countdown-label" class="caps">Time to apply</div>
          <p>Applications are open to join this fundraising round. If you're working on anything related to Eth2, we'd love to hear about your project.</p>
          <div id="btn-row">
            <div id="view-criteria" class="link-primary">View criteria</div>
            <router-link to="/join"><div id="apply-to-join" class="btn-primary">Apply to join</div></router-link>
          </div>
        </div>
      </div>
      <div v-if="!$store.getters.hasContributionPhaseEnded && !isRoundFull" class="apply-callout desktop">
        <div class="column">
          <h2>Join the funding round</h2>
          <p>Add your project to the next funding round. If you're working on anything related to Eth2, you can join in.</p>
        </div>
        <div class="centered">
          <router-link to="/join"><div id="join-round" class="btn-primary">Join round</div></router-link>
        </div>
        <div class="centered">{{timeRemaining}} to join</div>
      </div>
```

**Comment by ryancreatescopy** [comment]
> @ryancreatescopy is it intentional that mobile & desktop have different content?
> See https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/Landing.vue#L17
> 
> ```
>         <div v-if="!$store.getters.hasContributionPhaseEnded && !isRoundFull" class="apply-callout mobile">
>           <div id="countdown" class="caps">11 days</div>
>           <div id="countdown-label" class="caps">Time to apply</div>
>           <p>Applications are open to join this fundraising round. If you're working on anything related to Eth2, we'd love to hear about your project.</p>
>           <div id="btn-row">
>             <div id="view-criteria" class="link-primary">View criteria</div>
>             <router-link to="/join"><div id="apply-to-join" class="btn-primary">Apply to join</div></router-link>
>           </div>
>         </div>
>       </div>
>       <div v-if="!$store.getters.hasContributionPhaseEnded && !isRoundFull" class="apply-callout desktop">
>         <div class="column">
>           <h2>Join the funding round</h2>
>           <p>Add your project to the next funding round. If you're working on anything related to Eth2, you can join in.</p>
>         </div>
>         <div class="centered">
>           <router-link to="/join"><div id="join-round" class="btn-primary">Join round</div></router-link>
>         </div>
>         <div class="centered">{{timeRemaining}} to join</div>
>       </div>
> ```

nope fixed in #143
